src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java

```diff
@@ -170,18 +170,29 @@ public CreateEntryResult importDocument(ParametersForImportDocument parameters)
                             if (httpResponse.getStatus() == 400 || httpResponse.getStatus() == 404 || httpResponse.getStatus() == 409 || httpResponse.getStatus() == 500) {
                                 try {
                                     String jsonString = new JSONObject(body).toString();
-                                    problemDetails = ProblemDetails.create(httpResponse.getStatus(), headersMap);
-                                    CreateEntryResult result = objectMapper.readValue(jsonString,
-                                            CreateEntryResult.class);
-                                    if (result != null && result.getOperations() != null) {
-                                        problemDetails
-                                                .getExtensions()
-                                                .put(CreateEntryResult.class.getSimpleName(), result);
-                                        String summary = ApiClientUtils.getCreateEntryResultSummary(result);
-                                        if (summary != null && !summary
-                                                .trim()
-                                                .isEmpty())
-                                            problemDetails.setTitle(summary);
+                                    if (httpResponse
```

Comment:
maybe it would be better to check if the content-type header has `application/problem+json` to determine if the response is problem details

Reply:
ok done